src/arplot.py

```diff
@@ -210,10 +210,12 @@ def plot_orderfits(slf, model, ydata, xdata=None, xmodl=None, textplt="Order", m
                 xmn = np.min(xmodl)
                 xmx = np.max(xmodl)
             else:
-                xmn = min(np.min(xtmp), np.min(xmodl))
-                xmx = max(np.max(xtmp), np.max(xmodl))
+                xmn = np.min(xtmp)
+                xmx = np.max(xtmp)
+                #xmn = min(np.min(xtmp), np.min(xmodl))
+                #xmx = max(np.max(xtmp), np.max(xmodl))
```

Comment:
But, if the model is above (or below) all data points, we won't see the model at all. In principle, unless the model is very bad, the model should go neatly through the data anyway. It's only when the model fails that this might be helpful.